install/libvips.js

```diff
@@ -156,6 +157,15 @@ try {
     if (fs.existsSync(tarPathCache)) {
       libvips.log(`Using cached ${tarPathCache}`);
       extractTarball(tarPathCache, platformAndArch);
+    } else if (localLibvipsDir) {
+      const tarPathLocal = path.join(path.resolve(localLibvipsDir), `v${minimumLibvipsVersionLabelled}`, tarFilename);
+      // If localLibvipsDir is given try to use binaries from local directory
+      if (fs.existsSync(tarPathLocal)) {
```

Comment:
It might be simpler to remove the exists check and always call `extractTarball`. This means when it fails the error message from `createReadStream` will provide more context e.g. ENOENT vs EACCES etc., which I think will be more helpful.

Reply:
Yes, updated